lib/atlas/initializer_input.rb

```diff
@@ -0,0 +1,23 @@
+module Atlas
+  class InitializerInput
+    include ActiveDocument
+
+    DIRECTORY = 'initializer_inputs'
+
+    attribute :query,           String
+    attribute :share_group,     Symbol
+    attribute :priority,        Integer, default: 0
+    attribute :unit,            String
+    attribute :user_value,      Float
```

Comment:
For normal inputs we do not have the value in them either. And we do not have the Input objects in the Scenario - only at runtime. Also, having getters for these in the Atlas classes does not help us - we need them in Etengine and that has its own Input class